catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveAllCommandTest.java

```diff
@@ -177,6 +217,66 @@ protected SolrCacheMBean getCacheProxy() {
         verify(mbean, times(numCatalogCalls)).removeAll();
     }
 
+    /**
+     * Checks a series of responses ending with "no" for the warning yes/no prompt
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testAccidentalRemoval() throws Exception {
+        // given
+        final Session mockSession = mock(Session.class);
```

Comment:
Looks like lines 228 to 249 is repeated 3 time with the only difference being  the return values. Could this be put in a separate private method and the inputs passed in from the tests?

Reply:
I chose to implement these unit tests this way because this unit test class need a lot of refactoring to begin with, but I guess I can go ahead and spend some time cleaning it up.